msteams-platform/concepts/messaging-extensions/create-extensions.md

```diff
@@ -110,6 +110,37 @@ To initiate actions from a  messaging extension set the `type` parameter to `act
 }
 ```
 
+### Initiate actions from messages
+
+> [!NOTE]
+> Initiating actions from messages is in [developer preview](~/resources/dev-preview/developer-preview-intro.md).
+
+In addition to initiating actions from the compose message area, you can also use your messaging extension to initiate an action from a message. This will allow you to send the contents of the message to your bot for processing, and optionally reply to that message with the results. Your users can access your messaging extension from the overflow `...` menu and then selecting `Take action` as in the image below.
+
+![Example of initiating an action from a message](~/assets/images/compose-extensions/messageextensions_messageaction.png)
+
+To enable your messaging extension to work from an action you'll need to add the `context` parameter to your messaging extension's `commands` object in your app manifest as in the example below. Valid strings for the `context` array are `"message"`, `"commandBar"`, and `"compose"`. The default value is `["compose", "commandBar"]`.
+
+```json
+"composeExtensions": [
+  {
+    "botId": "12345ab8-ab12-cd34-ef56-098abc123876",
+    "canUpdateConfiguration": true,
+    "commands": [
+      {
+        "id": "reassignTodo",
```

Comment:
Nit: I know it's just sample text, but let's have the id, description and title be consistent with each other.